src/uct/api/uct_def.h

```diff
@@ -210,6 +210,176 @@ typedef struct uct_cm_remote_data {
 } uct_cm_remote_data_t;
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Server endpoint's connect callback arguments field mask.
+ *
+ * The enumeration allows specifying which fields in
+ * @ref uct_cm_ep_server_connect_cb_handle are present, for backward compatibility
+ * support.
+ */
+enum uct_cm_ep_server_connect_cb_handle_field {
+    /** Enables @ref uct_cm_ep_server_connect_cb_handle::status */
```

Comment:
Indicates that ... is valid ...

Reply:
Instead of 
/** Enables @ref uct_cm_ep_server_connect_cb_handle::status */
?
this is how we document the bits everywhere in ucx api... https://github.com/openucx/ucx/blob/master/src/uct/api/uct.h#L561

Reply:
this time the struct is passed from UCX to the user, not from user to UCX like in other APIs